neuro-cli/src/neuro_cli/blob_storage.py

```diff
@@ -140,6 +164,7 @@ async def mkbucket(
     name: Optional[str] = None,
     cluster: Optional[str] = None,
     org: Optional[str] = None,
+    project: Optional[str] = None,
```

Comment:
minor: do we forbid the creation of buckets with the same name in different projects?
currently, I cannot do it and it goes from the API.

Reply:
Should be possible. When did you test this last time?

Reply:
I think I found a bug in buckets API, will work on this, thank you